todolist/settings.py

```diff
@@ -63,8 +63,12 @@
 
 DATABASES = {
     "default": {
-        "ENGINE": "django.db.backends.sqlite3",
-        "NAME": os.path.join(BASE_DIR, "db.sqlite3"),
+        'ENGINE': 'mysql.connector.django',
+        'NAME': 'app_db',
+        'USER': 'app_user',
+        'PASSWORD': '1234',
+        'HOST': '172.17.0.3',
```

Comment:
The `HOST` value for the database is set to a specific IP address. Ensure that this IP address is correct and accessible from the Django application. Consider using environment variables to manage such configurations.